### PR TITLE
Dutch improvements

### PR DESCRIPTION
**Pull request by Jacco**
fixes: #31 

- passing options to constuctor
- trimRight end result
- passing third parameter of tests to n2words as options
- many dutch tests added
- formatted cards for readability
- default options / localOptions for NL
- use 'ën' in stead of 'en' after vowel under 100
- between 1100 and 9999 dutch uses 'HundredPairs'
- added option to opt out noHundredPairs (both spellings are fine)
- noHundredPairs defaults to false
- added includeOptionalAnd option (uncommon)
- specifying no option should render 'common' language

### CODE REVIEW

**Comment by Jacco** [comment]
@jvanaert are you OK with my suggested changes to the support of dutch?

@TylerVigario is the way I use options to change behaviour OK with you?

**Comment by jvanaert** [comment]
very good improvements to have! ❤️

**Comment by forzagreen** [comment]
Thanks a lot @Jacco for your contribution ! 👍 
It looks good to me.

Concerning the `localOptions`, it's ok since it's optional.
In future, if we have many languages with `localOptions`, we will document its usage in the README. For now, there is no need to document this special case.

I'll merge soon and release to npm ;)

**Comment by Jacco** [comment]
@forzagreen thx :-) Happy to contribute.

Also have an idea for implementing ordinals:
[https://en.wikipedia.org/wiki/Ordinal_numeral](https://en.wikipedia.org/wiki/Ordinal_numeral)

Would you want that feature in this library?

**Comment by forzagreen** [comment]
Yes @Jacco . It's planned to implement ordinal numbers.
For information, this package is largely inspired from python `num2words`

File: src/i18n/NL.mjs [hunk 1]
```diff
@@ -1,9 +1,56 @@
...
+export default function(options) {
   Num2Word_Base.call(this);
 
-  this.cards = [{'1000000000000000000000000000': 'quadriljard'}, {'1000000000000000000000000': 'quadriljoen'}, {'1000000000000000000000': 'triljard'}, {1000000000000000000: 'triljoen'}, {1000000000000000: 'biljard'}, {1000000000000: 'biljoen'}, {1000000000: 'miljard'}, {1000000: 'miljoen'}, {1000: 'duizend'}, {100: 'honderd'}, {90: 'negentig'}, {80: 'tachtig'}, {70: 'zeventig'}, {60: 'zestig'}, {50: 'vijftig'}, {40: 'veertig'}, {30: 'dertig'}, {20: 'twintig'}, {19: 'negentien'}, {18: 'achtien'}, {17: 'zeventien'}, {16: 'zestien'}, {15: 'vijftien'}, {14: 'veertien'}, {13: 'dertien'}, {12: 'twaalf'}, {11: 'elf'}, {10: 'tien'}, {9: 'negen'}, {8: 'acht'}, {7: 'zeven'}, {6: 'zes'}, {5: 'vijf'}, {4: 'vier'}, {3: 'drie'}, {2: 'twee'}, {1: 'een'}, {0: 'nul'}];
+  const defaultOptions = {
+    includeOptionalAnd: false,
+    noHundredPairs: false,
+  };
+
+  const localOptions = Object.assign({}, defaultOptions, options);
```
Comment (TylerVigario) [inline src/i18n/NL.mjs:11]: No need for an empty object in the first param.

To shorten everything up I usually do something like this.

```javascript
export default function(options) {
  options = Object.assign({
    includeOptionalAnd: false,
    ...
  }, options);
}
```

**Pull request merged**